tab-command/src/service/main/list_tabs.rs

```diff
@@ -53,10 +60,27 @@ impl MainListTabsService {
         let target_len = len + 4;
 
         println!("Available tabs:");
+        let cwd: String = env::current_dir()
+            .unwrap_or_else(|_| {
+                let mut path = std::path::PathBuf::new();
+                path.push("/");
+                path
+            })
+            .to_str()
+            .unwrap_or_default()
+            .to_owned();
 
         for tab in tabs.iter() {
             let name = &tab.name;
-            print!("    {}", name);
+            print!("    ");
+
+            if is_active(tab, &cwd) && *name == get_working_tab() {
```

Comment:
I think this first condition could just check the TAB env var.  It's fine to display the color if an anonymous tab (e.g. no yaml config) is selected.